ansible/roles/discourse/templates/standalone.yml.j2

```diff
@@ -52,7 +52,7 @@ env:
 
   ## Uncomment if you want the container to be started with the same
   ## hostname (-h option) as specified above (default "$hostname-$config")
-  DOCKER_USE_HOSTNAME: true
+  DOCKER_USE_HOSTNAME: false
```

Comment:
I don't get why you've added this but I guess it is only aesthetics. (I've read https://github.com/discourse/discourse_docker/pull/202)

Reply:
AFAIK there is no need for the container to have the same hostname as the host.